mapstory/templates/maps/map_detail.html

```diff
@@ -29,7 +29,7 @@
   {% autoescape off %}
       $('<iframe>', {
      {% if resource.chapters %}
-     src: '{%  url 'mapstory_view' resource.id %}',
+     src: '{%  url 'mapstory_view' resource.slug %}',
```

Comment:
Can we prevent this from making `/story/None/view` on existing stories that only have an ID?